test/extended/cli/observe.go

```diff
@@ -26,7 +26,7 @@ var _ = g.Describe("[sig-cli] oc observe", func() {
 
 		out, err = oc.Run("observe").Args("daemonsets", "--once").Output()
 		o.Expect(err).NotTo(o.HaveOccurred())
-		o.Expect(out).To(o.ContainSubstring("Nothing to sync, exiting immediately"))
+		o.Expect(out).To(o.Or(o.ContainSubstring("Sync ended"), o.ContainSubstring("Nothing to sync, exiting immediately")))
```

Comment:
It's not clear to me why we need this change here?

Reply:
In general case, `oc observe daemonsets --once` might successfully be synced and command can return `Sync ended` and this needs to be captured in test. For example, we have a daemonset, namely image-cache, and when we run this command, it returns `Sync ended` instead raising an error `Nothing to sync, exiting immediately`.